onbuild/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM finntech/node:6.9.1
+
+# All but package.json is optional
+ONBUILD COPY package.json yarn.lock* .npmrc* npm-shrinkwrap.json* ./
+
+# Install dependencies for native builds
+# This is in one giant command to keep the image size small
+ONBUILD RUN apk add --no-cache --virtual build-dependencies make gcc g++ python git && \
+    npm install --global yarn && \
+    # This script does `yarn install` if a `yarn.lock` file is present, otherwise `npm install`
+    install-dependencies.sh && \
+    npm cache clean && yarn cache clean && \
+    npm uninstall --global yarn && npm uninstall --global npm && \
+    apk del build-dependencies
```

Comment:
Thinking again, I think bundling a webapp should be handled outside of creating an image. The only reason we do install inside of the image is because of native dependencies.

So your workflow should be

```sh
npm it
npm run compile # (or something)
docker build -t my-app .
```

Which then copies over the compiled assets (and `.dockerignore` has your source code in it).

A bundled/transpiled app doesn't have native deps in it either way, so that shouldn't matter